README.md

```diff
@@ -25,18 +25,16 @@ Check out the [examples](examples) directory.
 
 1. Install the library
 
-        git clone https://github.com/jupyter-incubator/sparkmagic
-        cd sparkmagic
-        pip install -e .
+        pip install sparkmagic
 
 2. (Optional) Install the wrapper kernels
 
-        jupyter-kernelspec install remotespark/kernels/sparkkernel
-        jupyter-kernelspec install remotespark/kernels/pysparkkernel
+        jupyter-kernelspec install sparkmagic/kernels/sparkkernel
```

Comment:
It doesn't work.

    [Anaconda2] C:\Users\ristew>jupyter-kernelspec install sparkmagic/kernels/sparkkernel
    Traceback (most recent call last):
       File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\Scripts\jupyter-kernelspec-script.py", line 7, in <module>
        main()
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\Scripts\jupyter-kernelspec-script.py", line 4, in main
        KernelSpecApp.launch_instance()
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\lib\site-packages\traitlets\config\application.py", line 589, in launch_instance
        app.start()
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\lib\site-packages\jupyter_client\kernelspecapp.py", line 269, in start
        return self.subapp.start()
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\lib\site-packages\jupyter_client\kernelspecapp.py", line 141, in start
        replace=self.replace,
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\lib\site-packages\jupyter_client\kernelspec.py", line 269, in install_kernel_spec
        shutil.copytree(source_dir, destination)
      File "C:\Users\ristew\AppData\Local\Continuum\Anaconda2\lib\shutil.py", line 171, in copytree
        names = os.listdir(src)
    WindowsError: [Error 3] The system cannot find the path specified: u'sparkmagic/kernels/sparkkernel\\*.*'